docs/content/practices/state_clearing.md

```diff
@@ -92,22 +98,17 @@ adb shell am instrument -c TestClass#method2 -w com.package.name/junitRunnerClas
 adb pm clear
 ```
 
-Each test should be executed in an isolated instrumented process and junit reports should be merged into a big one
-report when all tests are finished.
-
-That's the common idea of `Orchestrator`.
+That's the idea behind of `Orchestrator`.
 <br/>
-It's just an `apk` which consist of
-only [several classes](https://github.com/android/android-test/tree/master/runner/android_test_orchestrator/java/androidx/test/orchestrator)
-and runs tests and clears data, as described above.
+It's an `apk` which only consists of [several classes](https://github.com/android/android-test/tree/master/runner/android_test_orchestrator/java/androidx/test/orchestrator)
+that run tests and clear data, as described above.
 
 You should install an `orchestrator` along with `application.apk` and `instrumented.apk` on the device.
 
-However, it's not the end.
+But that's not all.
 <br/>
-Orchestrator should somehow execute adb commands. Under the hood, it
-uses [special services.](https://github.com/android/android-test/tree/master/services)
-It's just a shell client and should be installed to the device.
+Orchestrator also needs to execute adb commands. For that it uses [special services.](https://github.com/android/android-test/tree/master/services) under the hood.
+It's just a shell client and should be installed on the device.
```

Comment:
After reading this, I was doubting whether I have to install it on my own, or the Orchestrator does it for me (this one right?). This should be clarified